docs/reference/integrations.md

```diff
@@ -0,0 +1,66 @@
+# Integrations
+
+### db
+
+_Interface_: mysql  
+_Supported charms_: [charmed-osm-mariadb-k8s](https://charmhub.io/mysql),
+[mysql-k8s](https://charmhub.io/mysql-k8s)
+
+Database integration is a required relation for the wordpress-k8s charm to supply structured data
+storage for WordPress. It is recommended to use a juju native integration that provides a mysql
+interface by providing necessary configuration details to wordpress-k8s charm. See
+[configuration](configuration.md) for more specific parameters.
```

Comment:
Will this link [configuration](configuration.md) work on charmhub?

Reply:
Updated to the configure tab link: https://charmhub.io/wordpress-k8s/configure?channel=edge
Thank you for the comment!

Reply:
Once it is on stable it might be confusing to get a link to edge

Reply:
Would it be better if the last documentation PR includes a cleanup of all the link pointing to edge so that the edge documentation can be easily confirmed with navigation before publishing to stable?
Otherwise, I'm happy to remove the edge channel link :)

Reply:
Sounds like a reasonable approach, although it would be good to get to stable sooner rather than later